pipeline/restart_node.sh

```diff
@@ -241,6 +241,53 @@ probe_node_type() {
 	explorer_shard="${3}"
 }
 
+probe_node_type_new() {
+	local output
+	if ! output="$(node_ssh "${ip}" '
+		if [ "X$(sudo systemctl is-enabled harmony.service)" = "Xenabled" ]
```

Comment:
Could write it like this as well: `if sudo systemctl is-enabled --quiet harmony.service`